docs/source/security/terminating-ssl.mdx

```diff
@@ -4,24 +4,25 @@ title: Terminating SSL
 
 Most production environments use a load balancer or HTTP proxy (such as nginx) to perform SSL termination on behalf of web applications in that environment.
```

Comment:
this is another doc page that feels unnecessary (there is literally nothing Apollo Server specific about it and running your own https from within Node is an odd thing to do) but maybe it has helped one person ever 🤷

Reply:
😂 So I'll leave it for now? Or would you rather remove?

Reply:
I mean the good news is that it's so unrelated to Apollo Server that we don't break the code example too often... So I guess leave it.